src/pages/Home.js

```diff
@@ -1,12 +1,13 @@
 import { useState } from "react";
 import { NavLink } from "react-router-dom";
+import moment from "moment";
```

Comment:
Moment is a huge package and adds a lot of weight to your project.
Please use a lightweight database like date-fns.